Model/DataObject/ClassDefinition/Data/ObjectBridge.php

```diff
@@ -1371,4 +1378,36 @@ public function preGetData($object, $params = [])
             return $data;
         }
     }
+
+    /**
```

Comment:
Possible to add a deprecation warning?

So that in de future we know that we have to remove this.